pages/login.tsx

```diff
@@ -42,7 +41,7 @@ type LoginProps = {
 export default function Login({ showNotification, loginContext }: LoginProps) {
   const $router = useRouter()
   const { status } = useProtectedRoute({
-    ifLoggedIn: `/leaderboard?toast=${btoa("You're already logged in.")}`,
+    ifLoggedIn: `/dashboard?toast=${btoa("You're already logged in.")}`,
```

Comment:
If we are removing login page why is it sending to `/dashboard` instead of `/leaderboard`? Isn't dashboard only for if login is enabled?

Reply:
We are removing sign up page